README.md

```diff
@@ -1,45 +1,36 @@
+A Tethered Ring is a dynamic structure that can cost-effectively support space launch facilities, transportation infrastructure, and a sizable human settlement at high altitudes. An altitude of 32 km and a population of 250,000 is shown to be achievable using circa 2017 science and technology.  The structure resembles a pipeline formed into a ring having a diameter similar to that of Earth’s moon. It stays aloft primarily by generating and properly combining inertial forces with tensile forces to offset the pull of gravity. The inertial forces are produced by the circular motion of magnetically levitated rings within the pipeline, and the tensile forces are generated by appropriately tethering the pipelines to the planet using stays made of strong, light-weight, industrial fiber.  The entire structure can be fabricated on Earth and neither its construction nor its deployment depends on a pre-existing space infrastructure or space-based industry. The deployed structure is resilient to catastrophic failure because its precision-guided fast moving components are: a) not exposed to seismic or climatic battering, and b) safely above and thus out of range of tacit civilizational threats, such attacks involving torpedoes or commandeered aircraft.  The tethered ring is an optimal “stepping stone” infrastructure for furnishing humanity with a safe, affordable, and sustainable means to escape Earth’s gravity, expand its civilization into space, and ultimately evolve into a multi-planetary species.  Check out the brief explanation and full whitepaper at: https://www.project-atlantis.com
 
 # Getting started
-1.	If you haven't already, install the 16.13.0 LTS version of node.js (note: a more recent version may work too) from here: https://nodejs.org/en/
-2.	Create a directory for three.js projects on your PC. For example: C:\Users\Username\Documents\repos\Three.js\
-3.	Clone the latest version of the TetheredRing project into this directory
-```
-PS C:\Users\Username\Documents\repos\Three.js> git clone https://github.com/philipswan/TetheredRing.git
-```
-4.	If on Windows, open “PowerShell” and “cd” to the directory that you created is step 2. Then run the following commands in that directory (only use the text after the command prompts below):
+
+- **Install node (19.0.1 tested) & npm (9.1.1 tested)**
```

Comment:
I prefer to be really explicate about what steps the user needs to take, and not assume any skills or fore-knowledge. I've had the experience of instructions not working because the author wasn't crystal clear about what directory each command needed to be run in.
Its also useful to occasionally include checkpoint instructions such as "Verify that you are on version <number> of <tool> by entering <command>".